pkg/apis/serving/v1alpha1/knativeserving_types.go

```diff
@@ -60,6 +60,11 @@ type KnativeServingSpec struct {
 	// If no registry is provided, the knative release images will be used.
 	// +optional
 	Registry Registry `json:"registry,omitempty"`
+
+	// A means to prevent installation of the knative services.
+	// By default knative serving will be installed. Setting ignore to true, will cause the operator not to install knative serving.
+	// +optional
+	Ignore bool `json:"ignore,omitempty"`
```

Comment:
So if we set Ignore to false, it will be equivalent to skipping this field?

Reply:
Correct. The default is false. I don't want there to be a difference between a no value and a false. I don't know if go will support a trinary of nil,true,false.

I imagine the usage will be either provide "ignore: true" or leave it out entirely.

Reply:
If you want a trinary value, you need to do something like [corev1.ConditionStatus](https://godoc.org/k8s.io/api/core/v1#ConditionStatus), I think.

Do we want the default to be "active" or "inactive"? If we want the default to be "active", then I think `Ignore` or `Disable` are good field names. If we want the default to be "inactive", then `Enable` or `Activate` are probably good names. (You want the zero-value `false` condition to line up with the name.)

I would probably slightly prefer `Disable` over `Ignore`, I think, but I leave it to your judgement.

Reply:
The only time I've found a trinary useful is when you really need to distinguish between "unknown" and "false". I don't think in this instance we want to distinguish between unknown and false, because according to the the discussion in https://github.com/knative/serving-operator/issues/39, the general consensus is that installing the operator should install knative by default. I think we want "unknown" to be the same as "install knative serving". Deploy the CRD and the operator and then knative is installed.